languages.yaml

```diff
@@ -1639,6 +1639,36 @@ OCaml:
     ansi:
       - yellow
     chip: "#3BE133"
+Odin:
+  type: programming
+  ascii: |
+    {0}                 @@@@
+    {0}         @@@@@@@{1}@@@@@    @@@@
+    {0}      @@@@@@    {1}@@@@    @@@@@@@{2}@
+    {0}   @@@@@      {1}@@@@    @@@@ {2}   @@@@
+    {0}  @@@@       {1}@@@@    @@@@ {2}      @@@@
+    {0} @@@       {1}@@@@@   @@@@ {2}         @@@@
+    {0}@@@       {1}@@@@    @@@@ {2}           @@@@
+    {0}@@@      {1}@@@@   @@@@ {2}              @@@
+    {0}@@@    {1}@@@@    @@@@ {2}               @@@
+    {0}@@@   {1}@@@@   @@@@ {2}                 @@@
+    {0}@@@{1}@@@@@    @@@@ {2}                 @@@@
+    {1} @@@@@@   @@@@@ {2}                 @@@@
+    {1}   @@    @@@@  {2}                @@@@
+    {1}        @@@@   {2}             @@@@@
+    {1}       @@@@@@@{2}@@      @@@@@@@@
+    {1}            @{2}@@@@@@@@@@@@@@
+  colors:
+    ansi:
+      - blue
+      - blue
+      - blue
+    hex:
+      - "#64B9FA"
+      - "#189CFF"
+      - "#107BCC"
+    chip: "#60AFFE"
```

Comment:
Great first design @spsandwichman, I took some time to clean up your ASCII art to make it more symmetrical and I also added a background:

![image](https://github.com/o2sh/onefetch/assets/13710835/3911f7b3-7d53-4d38-841a-378926b259f1)


I'd suggest you follow @spenserblack's suggestion and apply the gradient to the background


```
Odin:
  type: programming
  ascii: |
    {0}@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
    {0}@@@@@@@@@@@@@@@{1}@@@@@@@@@{0}@@@@@@@@@@@@@@@@
    {0}@@@@@@@@@@@{1}@@@@{0}@@@@@{1}@@@{0}@@@{1}@@@@{0}@@@@@@@@@@
    {0}@@@@@@@@{1}@@@{0}@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@{1}@@@{0}@@@@@@@@
    {0}@@@@@@{1}@@@{0}@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@{1}@@@{0}@@@@@@
    {0}@@@@{1}@@@{0}@@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@{1}@@@{0}@@@@
    {0}@@@{1}@@@{0}@@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@{1}@@@{0}@@@
    {0}@@{1}@@@{0}@@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@{1}@@@{0}@@@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@@{1}@@@{0}@@
    {0}@@@{1}@@@{0}@@@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@@{1}@@@{0}@@@
    {0}@@@@{1}@@@{0}@@{1}@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@@{1}@@@{0}@@@@
    {0}@@@@@@{1}@@@@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@@{1}@@@{0}@@@@@@
    {0}@@@@@@@@{1}@@{0}@@@{1}@@@{0}@@@@@@@@@@@@@{1}@@@{0}@@@@@@@@
    {0}@@@@@@@@@@@@{1}@@@{0}@@@@@@@@@@{1}@@@@{0}@@@@@@@@@@@
    {0}@@@@@@@@@@@@@@@{1}@@@@@@@@@@{0}@@@@@@@@@@@@@@@
    {0}@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
  colors:
    ansi:
      - blue
      - white
    hex:
      - "#3882D2"
      - "#FFFFFF"
    chip: "#60AFFE"
```

Reply:
Here is a second revision:

![image](https://github.com/o2sh/onefetch/assets/13710835/1790a5c1-0131-4387-bbc1-c34e8ba49f78)

Thinking about it, incorporating the gradient to the background might be challenging, It's up to you @spsandwichman if you want to try.


```
Odin:
  type: programming
  ascii: |
    {0}oooooooooooooooooooooooooooooooooooo
    {0}oooooooooooooooooooooooooooooooooooo
    {0}ooooooooooooo{1}@@@@@@@@@{0}oooooooooooooo
    {0}ooooooooo{1}@@@@{0}ooooo{1}@@@{0}ooo{1}@@@{0}ooooooooo
    {0}ooooooo{1}@@@@{0}oooooo{1}@@@{0}ooo{1}@@@@@@{0}ooooooo
    {0}ooooo{1}@@@@{0}ooooooo{1}@@@{0}ooo{1}@@@{0}oo{1}@@@@{0}ooooo
    {0}oooo{1}@@@{0}oooooooo{1}@@@{0}ooo{1}@@@{0}ooooo{1}@@@{0}oooo
    {0}ooo{1}@@@{0}oooooooo{1}@@@{0}ooo{1}@@@{0}ooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}ooooooo{1}@@@{0}ooo{1}@@@{0}oooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}oooooo{1}@@@{0}ooo{1}@@@{0}ooooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}ooooo{1}@@@{0}ooo{1}@@@{0}oooooooooo{1}@@@{0}ooo
    {0}ooo{1}@@@{0}oooo{1}@@@{0}ooo{1}@@@{0}ooooooooooo{1}@@@{0}ooo
    {0}oooo{1}@@@{0}oo{1}@@@{0}ooo{1}@@@{0}ooooooooooo{1}@@@{0}oooo
    {0}ooooo{1}@@@@@@{0}ooo{1}@@@{0}oooooooooo{1}@@@@{0}ooooo
    {0}ooooooo{1}@@@{0}ooo{1}@@@{0}ooooooooo{1}@@@@{0}ooooooo
    {0}oooooooooooo{1}@@@{0}oooooooo{1}@@@@{0}ooooooooo
    {0}ooooooooooooo{1}@@@@@@@@@@{0}ooooooooooooo
    {0}oooooooooooooooooooooooooooooooooooo
    {0}oooooooooooooooooooooooooooooooooooo
  colors:
    ansi:
      - blue
      - white
    hex:
      - "#3882D2"
      - "#FFFFFF"
    chip: "#60AFFE"
```

Reply:
would you mind if i removed the background and implemented a blue gradient with the O itself? I personally like the no-background look, but it's totally your call, and thank you!

Reply:
Of course go ahead 👍  Take the second revision as a starting to point for that and remove the BG and apply the gradient to the logo itself